ethereum/core/src/main/java/tech/pegasys/teku/core/signatures/Signer.java

```diff
@@ -52,4 +54,6 @@
       ContributionAndProof contributionAndProof, ForkInfo forkInfo);
 
   boolean isLocal();
+
+  Optional<URL> getSigningServiceUrl();
```

Comment:
@ajsutton we'll need a signer url for external signers, just checking you're happy with us adding this to the signer...

Reply:
Yeah that's fine.  I wonder if we should then redefine `isLocal()` to be a default method that returns true if there is a signer service URL rather than overriding it in different implementations?